src/app/find/errorReports/ErrorReportContainer.jsx

```diff
@@ -0,0 +1,187 @@
+/* eslint-disable max-len */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import { Route, Redirect, Switch, BrowserRouter as Router } from 'react-router-dom';
+import { postErrorReport, getLocation } from '../../../actions';
+import { selectLocationError, selectLocationData } from '../../../selectors/location';
+import LoadingLabel from '../../../components/form/LoadingLabel';
+import NotFound from '../../notFound/NotFound';
+import ErrorLabel from '../../../components/form/ErrorLabel';
+import Modal from '../../../components/modal';
+import Icon from '../../../components/icon';
+import ErrorReportText from './ErrorReportText';
+import ErrorReportInformationSelect from './ErrorReportInformationSelect';
+import Thanks from './Thanks';
+
+const LoadingView = ({ message }) => (
+  <div className="m-4">
+    <LoadingLabel>
+      {message}
+    </LoadingLabel>
+  </div>
+);
+
+class ErrorReportContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      errorReportInformation: {
+        errorReportGeneralLocationError: false,
```

Comment:
Since you're initializing this with valid errorReportGeneralLocationError and errorReportServices values, didn't you say that means you can't distinguish the case where no data has been entered from the one where it was entered, and was just empty?

Well, anyway, it's fine for now. Just means that if a user refreshes while in this flow, they'll submit without knowing their checkbox selections are wiped, and we'll get the data as if they left them empty. But that should be a rare case, hopefully.

Reply:
Agree.
Yeah, the use-case where a user is currently navigating straight to `/thanks` or `/text` isn't being handled currently. I've got the object in place, and it should be initialized with null going forward so these checks can occur (I didn't get around to implementing the logic).